i5k/settings.py

```diff
@@ -367,7 +366,7 @@
         },
         'i5k_file': {
             'class': 'logging.handlers.TimedRotatingFileHandler',
-            'filename': '/var/log/i5k/i5k.log',
+            'filename': '/var/log/django/i5k.log',
```

Comment:
This should be /var/log/i5k/i5k.log
There are 2 logging directories, 
 - /var/log/django is for django level logging
 - /var/log/i5k/ is for the app level logging